scannerazuresc.properties

```diff
@@ -6,8 +6,13 @@ organizationUrl=https://www.sonarsource.com
 homepageUrl=https://docs.sonarsource.com/sonarcloud/advanced-setup/ci-based-analysis/sonarcloud-extension-for-azure-devops/
 issueTrackerUrl=https://sonarsource.atlassian.net/jira/software/c/projects/SONARAZDO/issues
 sourcesUrl=https://github.com/SonarSource/sonar-scanner-azdo
-archivedVersions=1.44.0,1.44.1,1.44.2,1.45.0,1.46.0,2.0.0,2.0.1
-publicVersions=2.2.0
+archivedVersions=1.44.0,1.44.1,1.44.2,1.45.0,1.46.0,2.0.0,2.0.1,2.2.0
+publicVersions=2.3.1
+
+2.3.1.description=Embed Scanner for .NET 6.2.0.85879
```

Comment:
Is there anything we can say about what it brings? Mostly bug fixes from what I see.

Reply:
6.3.0 was supposed to bring "JVM no longer required".
6.3.1 reverted this change because of a bug in the Scanner .NET 7.0